Mathlib/Algebra/BigOperators/Fin.lean

```diff
@@ -7,6 +7,8 @@ import Mathlib.Data.Fintype.BigOperators
 import Mathlib.Data.Fintype.Fin
 import Mathlib.Data.List.FinRange
 import Mathlib.Logic.Equiv.Fin
+import Mathlib.Algebra.BigOperators.Finsupp
+import Mathlib.Data.Finsupp.Fin
```

Comment:
Hmmm... sorry for the iteration, but I think maybe `Mathlib.Algebra.BigOperators.Finsupp` is the better home after all

Reply:
Ok I have moved it, but your golf no longer compiles.

Reply:
Ok, nevermind it compiles when I move it to a different section.